packages/hooks/src/__tests__/useHasMFAccountDeposited.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { APIProvider, useFetch } from '@deriv/api';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useHasMFAccountDeposited from '../useHasMFAccountDeposited';
+
+jest.mock('@deriv/api', () => ({
+    ...jest.requireActual('@deriv/api'),
+    useFetch: jest.fn(() => ({ data: { get_account_status: { status: [] } } })),
+}));
+
+describe('useHasMFAccountDeposited', () => {
```

Comment:
@farhan-nurzi-deriv Please help to add more test cases to cover all the `expected_status` values 🙏🏻